README.md: minor fixes

lolmaus opened the pull request:
"Liquid Fire itself is an ember-cli application" was a *very* confusing statement.

ef4 commented:
Thanks. I agree about fixing that wording. But part of my point in including it was to point out that *this repo* is also a runnable ember-cli application. Maybe switch from "liquid fire itself" to "the liquid fire repo".

Also, `ember install liquid-fire` was not a mistake, it's supposed to be the normal way to install addons since ember-cli 0.2.3.

lolmaus commented:
> But part of my point in including it was to point out that this repo is also a runnable ember-cli application.

I've adjusted the wording.


> Also, ember install liquid-fire was not a mistake, it's supposed to be the normal way to install addons since ember-cli 0.2.3.

I had tried running `ember install` with a version specified and it failed. But after you told me, i retried the same command (literally, by pressing Up in the terminal) and it worked. Probably some glitch.

I've reverted the command while still fixing the version number.

ef4 commented:
Thanks.
